src/Api/MailingList/Member.php

```diff
@@ -204,9 +207,12 @@ public function update(string $list, string $address, array $parameters = [])
                     // We should assert that "vars"'s $value is a string.
                     // no break
                 case 'address':
-                case 'name':
                     Assert::stringNotEmpty($value);
 
+                    break;
+                case 'name':
+                    Assert::nullOrStringNotEmpty($value);
```

Comment:
Based on the condition below...
```php
        if (array_key_exists('name', $parameters) && null === $parameters['name']) {
            unset($parameters['name']);
        }
```
don't you think that's better to check if it's a non-empty string instead of allow `null` and then unset the key?

Reply:
Null is allowed value for member name in Mailgun